src/lib/domHandler.js

```diff
@@ -0,0 +1,20 @@
+
+function updateUi (data){
+  document.getElementById('weather-report').innerHTML =  `
```

Comment:
It's generally better to use `document.createElement` and `appendChild` than `innerHTML`. See [this](https://stackoverflow.com/questions/11515383/why-is-element-innerhtml-bad-code)